src/Symfony/Component/PropertyInfo/Extractors/DoctrineExtractor.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ * (c) Kévin Dunglas <dunglas@gmail.com>
```

Comment:
Not sure if it is required or not as this code was in the wild previously but I'm ok to remove this line if necessary.